S21/psriraj17/Code_review.py

```diff
@@ -31,13 +31,13 @@ def Greek(word):
 
 if __name__ == '__main__':
 
- lang = input("Enter your language (insert either symbol or language name):  ")
- str  = input("Enter your String:  " )
+ lang = input("Enter your language (insert either symbol or language name):  ") #User input of language
+ str  = input("Enter your String:  " ) # userinput for word
  if lang== "Turkish" or lang== "turkish" or lang== "tr":
-    turkish(str)
+    turkish(str) #function call 
  if lang== "Irish" or lang== "ga" or lang== "irish" or lang== "ga-IE":
-     irish(str)
+     irish(str)# function call
  if lang== "Greek" or lang== "el" or lang== "greek":
-     Greek(str)
+     Greek(str)# function call
 else:
      print(str.lower())

```

Comment:
I think this for the Other languages like chinese,  japanese, to show the output for test cases, but its not working.
Could you please check that?

Reply:
Yeah, code is excellent and reduced the space and time complexity with good implementation.
The test cases for 3 languages is good.
Just needed to be sure to check for remaining languages.
And
Good Work  Raj (psriraj17)

Reply:
The Test Cases output is correct for Irish, Greek, Turkish.